pkg/scheduler/framework/plugins/interpodaffinity/plugin.go

```diff
@@ -135,6 +135,6 @@ func GetNamespaceLabelsSnapshot(ns string, nsLister listersv1.NamespaceLister) (
 		// Create and return snapshot of the labels.
 		return labels.Merge(podNS.Labels, nil)
 	}
-	klog.ErrorS(err, "getting namespace, assuming empty set of namespace labels", "namespace", ns)
+	klog.V(3).InfoS("getting namespace, assuming empty set of namespace labels", "namespace", ns, "err", err)
```

Comment:
I think this should be error logs because the namespace of Pod should be gotten successfully. 
And, either way, we can avoid this log by creating a namespace for a Pod in scheduler_perf. 
I created PR for this: #107700
@alculquicondor  Please take a look at this 🙇

Reply:
If it's an error, then the function should return an error and whoever calls it needs to deal with it (abort the operation or log the error and proceed, depending on what's appropriate).

Reply:
Yeah, I also like the way you are saying; return error from this kind of utility func and log error (or abort scheduling) in who calls it. 
I still think it should be error log, but it is not a strong opinion. Anyway, it is not big deal here cuz #107700 will resolve this error log.

Reply:
I briefly looked at what it take to return the error, but it gets called in a few places and I don't know any of them, so I'd prefer to not touch them.

Reply:
Odd ordering of messages here :grin: 

There''s no particular reason for `V(3)`. It's defined as "Extended information about changes". `V(2)` is the recommended default ("Useful steady state information about the service and important log messages that may correlate to significant changes in the system. ").

Shall I change it to `V(2)`?

Reply:
/hold

Reply:
I see. Where are those definitions?

It sounds like it fits 3 better.

Reply:
From https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md#logging-conventions

I've kept the level and updated the message as you suggested. Please LGTM again so that this can be merged once we have approval.

Reply:
/hold cancel